cime_config/machines/cmake_macros/gnu_melvin.cmake

```diff
@@ -11,4 +11,3 @@ endif()
 if (MPILIB STREQUAL mpi-serial AND NOT compile_threaded)
   set(PFUNIT_PATH "$ENV{SEMS_PFUNIT_ROOT}")
 endif()
-string(APPEND SLIBS " -lblas -llapack")
```

Comment:
Is melvin still around? I thought it was no longer online.

Reply:
Yes, we badly need to go through all our configs and purge unused machines. I think we still have a lot of NCAR machine configs that we never use.